src/blocks/mail.rs

```diff
@@ -0,0 +1,105 @@
+use std::time::Duration;
+use chan::Sender;
+
+use block::{Block, ConfigBlock};
+use config::Config;
+use de::deserialize_duration;
+use errors::*;
+use widgets::text::TextWidget;
+use widget::{I3BarWidget, State};
+use input::I3BarEvent;
+use scheduler::Task;
+use maildir::*;
+
+use uuid::Uuid;
+
+pub struct Mail {
+    text: TextWidget,
+    id: String,
+    update_interval: Duration,
+
+    //useful, but optional
+    #[allow(dead_code)]
```

Comment:
There's no need to include these yet -- you might as well drop them and avoid the confusion of dead code.